x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/unsaved_timeline.cy.ts

```diff
@@ -38,10 +38,12 @@ import {
 } from '../../../tasks/timeline';
 import { EXPLORE_URL, hostsUrl, MANAGE_URL } from '../../../urls/navigation';
 
-// FLAKY: https://github.com/elastic/kibana/issues/169588
-describe.skip('Save Timeline Prompts', { tags: ['@ess'] }, () => {
-  before(() => {
+describe('Save Timeline Prompts', { tags: ['@ess'] }, () => {
```

Comment:
what do you think about having `ess` in the title, something like `Save timeline prompts - ess`? Right now both describe in this file have the exact same title...